daft/series.py

```diff
@@ -468,6 +468,11 @@ class SeriesImageNamespace(SeriesNamespace):
     def decode(self) -> Series:
         return Series._from_pyseries(self._series.image_decode())
 
+    def encode(self, image_format: str | ImageFormat) -> Series:
+        if isinstance(image_format, str):
+            image_format = ImageFormat.from_format_string(image_format.upper())
```

Comment:
Have an else case here if image_format isn't a str or ImageFormat. Otherwise we panic on the type extract in rust